es6-lib/config/config.js

```diff
@@ -5,7 +5,7 @@ default {
   //gate at which the service will not read the whole file to generate
   //a detailed summary. for files over this limit, an abbreviated summary
   //will be returned, which is much faster, but less helpful
-  abbreviateSummarySize: 500 * 1000, //500kb
+  abbreviateSummarySize: 1500 * 1000000, //1.5gb
```

Comment:
I worry a little about this number only being 1.5 times the nginx limit we have right now.

I might bump this even higher.